### ISSUE TITLE
Fix broken Netlify deploy

### ISSUE THREAD

**Issue by MasterKale** [MasterKale/SimpleWebAuthn#73]
For some reason after migrating the example project to TypeScript in PR #71 Netlify started having issues with `npm run docs`:

```sh
7:46:23 AM: Now using node v12.19.1 (npm v6.14.8)
# ...snip...
7:46:43 AM: $ npm run docs
# ...snip...
7:47:24 AM: Loaded plugin /opt/build/repo/typedoc-plugin-external-module-name
7:47:24 AM: Using TypeScript 3.9.7 from /opt/build/repo/node_modules/typescript/lib
7:47:32 AM: Error: /opt/build/repo/example/fido-conformance.ts(12)
7:47:32 AM:  Cannot find module '@simplewebauthn/server/dist/metadata/metadataService' or its corresponding type declarations.
7:47:32 AM: Error: /opt/build/repo/example/index.ts(11)
7:47:32 AM:  Cannot find module 'dotenv' or its corresponding type declarations.
7:47:32 AM: npm ERR! code ELIFECYCLE
7:47:32 AM: npm ERR! errno 4
7:47:32 AM: npm ERR! simplewebauthn-monorepo@ docs: `npm run bootstrap && typedoc --tsconfig tsconfigdoc.json`
7:47:32 AM: npm ERR! Exit status 4
7:47:32 AM: npm ERR!
7:47:32 AM: npm ERR! Failed at the simplewebauthn-monorepo@ docs script.
7:47:32 AM: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
7:47:32 AM: npm ERR! A complete log of this run can be found in:
7:47:32 AM: npm ERR!     /opt/buildhome/.npm/_logs/2020-11-17T15_47_32_501Z-debug.log
```
https://app.netlify.com/sites/simplewebauthn/deploys/5fb3f00102e84400084e2ef4 (I probably only have access to this right now)

I tried recreating this issue locally but docs generate just fine. I need to investigate this further to find out why it complains about the missing type declarations.

**Comment by MasterKale** [comment]
Aha! I was able to replicate this locally!

Node: v12.19.1
Commands:
```sh
$> CI=true npm ci
$> CI=true npm run docs
```

Progress...

**Comment by MasterKale** [comment]
Docs are generating again. The fix was delivered in PR #74
